# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: anaconda 23.7\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2015-05-15 17:45-0700\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"

#: pyanaconda/bootloader.py:248
msgid "/boot file system"
msgstr ""

#: pyanaconda/bootloader.py:405
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following raid "
"levels: %(raid_level)s."
msgstr ""

#: pyanaconda/bootloader.py:413
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following metadata "
"versions: %(metadata_versions)s."
msgstr ""

#: pyanaconda/bootloader.py:421
#, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following device "
"types: %(types)s."
msgstr ""

#: pyanaconda/bootloader.py:437
#, python-format
msgid "%(name)s must have one of the following disklabel types: %(types)s."
msgstr ""

#: pyanaconda/bootloader.py:449
#, python-format
msgid "%(desc)s cannot be of type %(type)s."
msgstr ""

#: pyanaconda/bootloader.py:455
#, python-format
msgid "%(desc)s must be mounted on one of %(mountpoints)s."
msgstr ""

#: pyanaconda/bootloader.py:467
#, python-format
msgid "%(desc)s must be between %(min)d and %(max)d MB in size"
msgstr ""

#: pyanaconda/bootloader.py:473
#, python-format
msgid "%(desc)s must not be smaller than %(min)dMB."
msgstr ""

#: pyanaconda/bootloader.py:482
#, python-format
msgid "%(desc)s must not be larger than %(max)dMB."
msgstr ""

#: pyanaconda/bootloader.py:500
#, python-format
msgid "%(desc)s must be within the first %(max_end)s of the disk."
msgstr ""

#: pyanaconda/bootloader.py:510
#, python-format
msgid "%s must be on a primary partition."
msgstr ""

#: pyanaconda/bootloader.py:646 pyanaconda/bootloader.py:746
#, python-format
msgid "%s cannot be on an encrypted block device."
msgstr ""

#: pyanaconda/bootloader.py:701
#, python-format
msgid "%s cannot be on an iSCSI disk on s390(x)"
msgstr ""

#: pyanaconda/bootloader.py:705
#, python-format
msgid "%(desc)s cannot be of type %(type)s"
msgstr ""

#: pyanaconda/bootloader.py:1375
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not. A drive failure in %(stage2dev)s "
"could render the system unbootable."
msgstr ""

#: pyanaconda/bootloader.py:1381
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not part of this array. The stage1 "
"boot loader will only be installed to a single drive."
msgstr ""

#: pyanaconda/bootloader.py:1670
#, python-format
msgid ""
"%(deviceName)s may not have enough space for grub2 to embed core.img when "
"using the %(fsType)s file system on %(deviceType)s"
msgstr ""

#: pyanaconda/constants.py:73
msgid "Start VNC"
msgstr ""

#: pyanaconda/constants.py:74
msgid "Use text mode"
msgstr ""

#: pyanaconda/constants.py:138
msgid "The password is empty."
msgstr ""

#: pyanaconda/constants.py:139
msgid "The passwords do not match."
msgstr ""

#: pyanaconda/constants.py:140
msgid "The passwords you entered were different.  Please try again."
msgstr ""

#: pyanaconda/constants.py:141
#, python-format
msgid "The password you have provided is weak. %s"
msgstr ""

#: pyanaconda/constants.py:142
#, python-format
msgid "The password you have provided is weak: %s."
msgstr ""

#: pyanaconda/constants.py:143
msgid "You have provided a weak password. Press Done again to use anyway."
msgstr ""

#: pyanaconda/constants.py:144
#, python-format
msgid "You have provided a weak password: %s. Press Done again to use anyway."
msgstr ""

#: pyanaconda/constants.py:145
msgid ""
"The password you have provided contains non-ASCII characters. You may not be "
"able to switch between keyboard layouts to login. Press Done to continue."
msgstr ""

#: pyanaconda/constants.py:146
msgid "You will have to press Done twice to confirm it."
msgstr ""

#: pyanaconda/constants.py:148
msgid "Empty"
msgstr ""

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:125
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:177
msgid "Weak"
msgstr ""

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:128
msgid "Fair"
msgstr ""

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:131
msgid "Good"
msgstr ""

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:134
msgid "Strong"
msgstr ""

#: pyanaconda/constants_text.py:40 pyanaconda/rescue.py:332
#: pyanaconda/rescue.py:364 pyanaconda/rescue.py:432 pyanaconda/rescue.py:448
#: pyanaconda/rescue.py:459 anaconda:517
msgid "OK"
msgstr ""

#: pyanaconda/constants_text.py:44 pyanaconda/text.py:52 pyanaconda/text.py:53
msgid "Cancel"
msgstr ""

#: pyanaconda/constants_text.py:48
msgid "Yes"
msgstr ""

#: pyanaconda/constants_text.py:52
msgid "No"
msgstr ""

#: pyanaconda/errors.py:126
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"The installation will now terminate."
msgstr ""

#: pyanaconda/errors.py:132
#, python-format
msgid "An error occurred while resizing the device %s."
msgstr ""

#: pyanaconda/errors.py:141
#, python-format
msgid ""
"There is a problem with your existing storage configuration: <b>"
"%(errortxt)s</b>\n"
"\n"
"You must resolve this matter before the installation can proceed. There is a "
"shell available for use which you can access by pressing ctrl-alt-f1 and "
"then ctrl-b 2.\n"
"\n"
"Once you have resolved the issue you can retry the storage scan. If you do "
"not fix it you will have to exit the installer."
msgstr ""

#: pyanaconda/errors.py:150 pyanaconda/ui/gui/main.glade:24
msgid "_Exit Installer"
msgstr ""

#: pyanaconda/errors.py:150
msgid "_Retry"
msgstr ""

#: pyanaconda/errors.py:157
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems.  Please check your hardware for the cause of this problem."
msgstr ""

#: pyanaconda/errors.py:166
msgid ""
"There is an entry in your /etc/fstab file that contains an invalid or "
"incorrect file system type:\n"
"\n"
msgstr ""

#: pyanaconda/errors.py:172
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error. Would you like to continue "
"using this image?"
msgstr ""

#: pyanaconda/errors.py:186
msgid ""
"The installer has tried to mount the installation image, but cannot find it "
"on the hard drive.\n"
"\n"
"Should I try again to locate the image?"
msgstr ""

#: pyanaconda/errors.py:196
#, python-format
msgid "An error occurred mounting the source device %s. Retry?"
msgstr ""

#: pyanaconda/errors.py:204
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""

#: pyanaconda/errors.py:212
#, python-format
msgid ""
"The group '%s' is required for this installation. This group does not exist. "
"This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:218
#, python-format
msgid ""
"You have specified that the group '%s' should be installed.  This group does "
"not exist.  Would you like to ignore this group and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:223
#, python-format
msgid ""
"You have specified that the group '%s' should be excluded from "
"installation.  This group does not exist.  Would you like to ignore this "
"group and continue with installation?"
msgstr ""

#: pyanaconda/errors.py:235
#, python-format
msgid ""
"The package '%s' is required for this installation. This package does not "
"exist. This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:242
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist.  Would you like to ignore this package and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:253
#, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  This is "
"a fatal error and installation will be aborted.  The details of this error "
"are:\n"
"\n"
"%(details)s"
msgstr ""

#: pyanaconda/errors.py:261
msgid ""
"The following error occurred while installing.  This is a fatal error and "
"installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:269
msgid ""
"The following software marked for installation has errors.  This is likely "
"caused by an error with\n"
"your installation source."
msgstr ""

#: pyanaconda/errors.py:277
msgid ""
"The following error occurred while installing the boot loader. The system "
"will not be bootable. Would you like to ignore this and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:289
#, python-format
msgid "Unable to encrypt password: unsupported algorithm %s"
msgstr ""

#: pyanaconda/errors.py:296
#, python-format
msgid ""
"Installation was stopped due to an error installing the boot loader. The "
"exact error message is:\n"
"\n"
"%s\n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/exception.py:79
#, python-format
msgid ""
"The installation was stopped due to what seems to be a problem with your "
"hardware. The exact error message is:\n"
"\n"
"%s.\n"
"\n"
" The installer will now terminate."
msgstr ""

#: pyanaconda/exception.py:83
msgid "Hardware error occured"
msgstr ""

#: pyanaconda/exception.py:138
#, python-format
msgid ""
"\n"
"The installation was stopped due to incomplete spokes detected while running "
"in non-interactive cmdline mode. Since there cannot be any questions in "
"cmdline mode, edit your kickstart file and retry installation.\n"
"The exact error message is: \n"
"\n"
"%s.\n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/exception.py:146
#, python-format
msgid ""
"\n"
"Running in cmdline mode, no interactive debugging allowed.\n"
"The exact error message is: \n"
"\n"
"%s.\n"
"\n"
"The installer will now terminate."
msgstr ""

#: pyanaconda/install.py:80
msgid "Configuring installed system"
msgstr ""

#: pyanaconda/install.py:93
msgid "Writing network configuration"
msgstr ""

#: pyanaconda/install.py:97
msgid "Creating users"
msgstr ""

#: pyanaconda/install.py:105
msgid "Configuring addons"
msgstr ""

#: pyanaconda/install.py:108
msgid "Generating initramfs"
msgstr ""

#: pyanaconda/install.py:119
#, python-format
msgid "Joining realm: %s"
msgstr ""

#: pyanaconda/install.py:122
msgid "Running post-installation scripts"
msgstr ""

#: pyanaconda/install.py:174
#, python-format
msgid "Waiting for %s threads to finish"
msgstr ""

#: pyanaconda/install.py:180
msgid "Setting up the installation environment"
msgstr ""

#: pyanaconda/install.py:207 pyanaconda/kickstart.py:2120
msgid "Running pre-installation scripts"
msgstr ""

#: pyanaconda/install.py:215
msgid "Discovering realm to join"
msgstr ""

#: pyanaconda/install.py:271
msgid "Installing boot loader"
msgstr ""

#: pyanaconda/install.py:274 pyanaconda/packaging/livepayload.py:178
#: pyanaconda/packaging/dnfpayload.py:659
msgid "Performing post-installation setup tasks"
msgstr ""

#: pyanaconda/iutil.py:815
msgid "No host url"
msgstr ""

#: pyanaconda/iutil.py:836
msgid "malformed URL, cannot parse it."
msgstr ""

#: pyanaconda/iutil.py:852
msgid "URL has no host component"
msgstr ""

#: pyanaconda/kickstart.py:164
#, python-format
msgid "Escrow certificate %s requires the network."
msgstr ""

#: pyanaconda/kickstart.py:172
#, python-format
msgid ""
"SSL error while downloading the escrow certificate:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/kickstart.py:175
#, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/kickstart.py:284
#, python-format
msgid "%s is missing. Cannot setup authentication."
msgstr ""

#: pyanaconda/kickstart.py:309
#, python-format
msgid "autopart fstype of %s is invalid."
msgstr ""

#: pyanaconda/kickstart.py:326
#, python-format
msgid "Settings default fstype to %s failed."
msgstr ""

#: pyanaconda/kickstart.py:357
msgid "GRUB2 does not support installation to a partition."
msgstr ""

#: pyanaconda/kickstart.py:417
#, python-format
msgid "More than one match found for given boot drive \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:420
#, python-format
msgid "Requested boot drive \"%s\" doesn't exist or cannot be used."
msgstr ""

#: pyanaconda/kickstart.py:462
#, python-format
msgid ""
"Btrfs partition \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"btrfs\"."
msgstr ""

#: pyanaconda/kickstart.py:467
#, python-format
msgid "Tried to use undefined partition \"%s\" in Btrfs volume specification."
msgstr ""

#: pyanaconda/kickstart.py:480
msgid ""
"Btrfs volume defined without any member devices.  Either specify member "
"devices or use --useexisting."
msgstr ""

#: pyanaconda/kickstart.py:489 pyanaconda/kickstart.py:862
#: pyanaconda/kickstart.py:1441
#, python-format
msgid "The mount point \"%s\" is not valid.  It must start with a /."
msgstr ""

#: pyanaconda/kickstart.py:504
#, python-format
msgid "Btrfs volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:590
#, python-format
msgid ""
"Disklabel \"%s\" given in clearpart command is not supported on this "
"platform."
msgstr ""

#: pyanaconda/kickstart.py:602
#, python-format
msgid "Disk \"%s\" given in clearpart command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:615
#, python-format
msgid "Device \"%s\" given in clearpart device list does not exist."
msgstr ""

#: pyanaconda/kickstart.py:642
#, python-format
msgid "NIC \"%s\" given in fcoe command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:693
#, python-format
msgid "%s is missing. Cannot setup firewall."
msgstr ""

#: pyanaconda/kickstart.py:741 pyanaconda/kickstart.py:752
#, python-format
msgid "Disk \"%s\" given in ignoredisk command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:765
#, python-format
msgid ""
"Network interface \"%(nic)s\" required by iSCSI \"%(iscsiTarget)s\" target "
"is not up."
msgstr ""

#: pyanaconda/kickstart.py:775
msgid ""
"iscsi --iface must be specified (binding used) either for all targets or for "
"none"
msgstr ""

#: pyanaconda/kickstart.py:868
#, python-format
msgid ""
"No volume group exists with the name \"%s\".  Specify volume groups before "
"logical volumes."
msgstr ""

#: pyanaconda/kickstart.py:875
#, python-format
msgid ""
"No thin pool exists with the name \"%s\". Specify thin pools before thin "
"volumes."
msgstr ""

#: pyanaconda/kickstart.py:883
msgid "logvol --noformat must also use the --name= option."
msgstr ""

#: pyanaconda/kickstart.py:888 pyanaconda/kickstart.py:949
#, python-format
msgid "Logical volume \"%s\" given in logvol command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:899 pyanaconda/kickstart.py:908
#: pyanaconda/kickstart.py:959 pyanaconda/kickstart.py:1218
#: pyanaconda/kickstart.py:1227 pyanaconda/kickstart.py:1309
#, python-format
msgid "Target size \"%(size)s\" for device \"%(device)s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:922
#, python-format
msgid ""
"Logical volume name \"%(logvol)s\" is already in use in volume group "
"\"%(volgroup)s\"."
msgstr ""

#: pyanaconda/kickstart.py:927
#, python-format
msgid ""
"Logical volume size \"%(logvolSize)s\" must be larger than the volume group "
"extent size of \"%(extentSize)s\"."
msgstr ""

#: pyanaconda/kickstart.py:938 pyanaconda/kickstart.py:1245
#: pyanaconda/kickstart.py:1491
#, python-format
msgid "The \"%s\" file system type is not supported."
msgstr ""

#: pyanaconda/kickstart.py:1032
msgid "No passphrase given for encrypted LV"
msgstr ""

#: pyanaconda/kickstart.py:1086 pyanaconda/kickstart.py:1090
#, python-format
msgid "The %s kickstart command is not currently supported."
msgstr ""

#: pyanaconda/kickstart.py:1115
#, python-format
msgid "No disk found for specified BIOS disk \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1150
#, python-format
msgid "RAID partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1161 pyanaconda/kickstart.py:1416
#, python-format
msgid "PV partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1172 pyanaconda/kickstart.py:1426
#, python-format
msgid "Btrfs partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1195
#, python-format
msgid "The size \"%s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:1202
msgid "part --noformat must also use the --onpart option."
msgstr ""

#: pyanaconda/kickstart.py:1207 pyanaconda/kickstart.py:1300
#, python-format
msgid "Partition \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1260 pyanaconda/kickstart.py:1274
#, python-format
msgid "Disk \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1263
#, python-format
msgid "Cannot install to unpartitionable device \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1270
#, python-format
msgid "Disk \"%s\" in part command is not partitioned."
msgstr ""

#: pyanaconda/kickstart.py:1283
#, python-format
msgid "The maximum size \"%s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:1353
msgid "No passphrase given for encrypted part"
msgstr ""

#: pyanaconda/kickstart.py:1448
msgid "raid --noformat must also use the --device option."
msgstr ""

#: pyanaconda/kickstart.py:1453
#, python-format
msgid "RAID device  \"%s\" given in raid command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1474
#, python-format
msgid ""
"RAID device \"%(device)s\" has a format of \"%(format)s\", but should have a "
"format of \"mdmember\"."
msgstr ""

#: pyanaconda/kickstart.py:1479
#, python-format
msgid "Tried to use undefined partition \"%s\" in RAID specification."
msgstr ""

#: pyanaconda/kickstart.py:1507
#, python-format
msgid "RAID volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1514
#, python-format
msgid "The RAID volume name \"%s\" is already in use."
msgstr ""

#: pyanaconda/kickstart.py:1751
#, python-format
msgid ""
"Physical volume \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"lvmpv\"."
msgstr ""

#: pyanaconda/kickstart.py:1756
#, python-format
msgid "Tried to use undefined partition \"%s\" in Volume Group specification"
msgstr ""

#: pyanaconda/kickstart.py:1762
#, python-format
msgid ""
"Volume group \"%s\" defined without any physical volumes.  Either specify "
"physical volumes or use --useexisting."
msgstr ""

#: pyanaconda/kickstart.py:1772
#, python-format
msgid ""
"Volume group given physical extent size of \"%(extentSize)s\", but must be "
"one of:\n"
"%(validExtentSizes)s."
msgstr ""

#: pyanaconda/kickstart.py:1779
msgid ""
"volgroup --noformat and volgroup --useexisting must also use the --name= "
"option."
msgstr ""

#: pyanaconda/kickstart.py:1784
#, python-format
msgid "Volume group \"%s\" given in volgroup command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1787
#, python-format
msgid "The volume group name \"%s\" is already in use."
msgstr ""

#: pyanaconda/kickstart.py:1843
msgid ""
"The upgrade kickstart command is no longer supported. Upgrade functionality "
"is provided through fedup."
msgstr ""

#: pyanaconda/network.py:98
msgid "Host name cannot be None or an empty string."
msgstr ""

#: pyanaconda/network.py:101
msgid "Host name must be 255 or fewer characters in length."
msgstr ""

#: pyanaconda/network.py:104
msgid ""
"Host names can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.', "
"parts between periods must contain something and cannot start or end with "
"'-'."
msgstr ""

#: pyanaconda/network.py:1325 pyanaconda/ui/gui/spokes/custom.py:544
#: pyanaconda/ui/gui/spokes/network.py:302
#: pyanaconda/ui/gui/spokes/lib/accordion.py:61
msgid "Unknown"
msgstr ""

#: pyanaconda/network.py:1329
msgid "Connecting..."
msgstr ""

#: pyanaconda/network.py:1331
msgid "Disconnecting..."
msgstr ""

#: pyanaconda/network.py:1351
#, python-format
msgid "Wired (%(interface_name)s) connected"
msgstr ""

#: pyanaconda/network.py:1354
#, python-format
msgid "Wireless connected to %(access_point)s"
msgstr ""

#: pyanaconda/network.py:1357
#, python-format
msgid "Bond %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1361
#, python-format
msgid "Team%(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1365
#, python-format
msgid "Bridge%(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1371
#, python-format
msgid "VLAN %(interface_name)s (%(parent_device)s, ID %(vlanid)s) connected"
msgstr ""

#: pyanaconda/network.py:1388
#, python-format
msgid "Connected: %(list_of_interface_names)s"
msgstr ""

#: pyanaconda/network.py:1391
msgid "Not connected"
msgstr ""

#: pyanaconda/network.py:1394 pyanaconda/ui/gui/spokes/network.glade:2097
msgid "No network devices available"
msgstr ""

#: pyanaconda/product.py:53
#, python-format
msgid "%(productName)s %(productVersion)s INSTALLATION"
msgstr ""

#: pyanaconda/product.py:57
#, python-format
msgid "New %(name)s %(version)s Installation"
msgstr ""

#: pyanaconda/pwpolicy.py:114
#, python-format
msgid "policy name required for %s"
msgstr ""

#: pyanaconda/pwpolicy.py:123
#, python-format
msgid "A %(command)s with the name %(policyName)s has already been defined."
msgstr ""

#: pyanaconda/rescue.py:183 pyanaconda/rescue.py:349 pyanaconda/rescue.py:425
#, python-format
msgid "Run %s to unmount the system when you are finished."
msgstr ""

#: pyanaconda/rescue.py:186
msgid "When finished please exit from the shell and your system will reboot."
msgstr ""

#: pyanaconda/rescue.py:199
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr ""

#: pyanaconda/rescue.py:287 pyanaconda/rescue.py:357 pyanaconda/rescue.py:428
#: pyanaconda/rescue.py:459
msgid "Rescue"
msgstr ""

#: pyanaconda/rescue.py:288
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.  \n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:298
msgid "Continue"
msgstr ""

#: pyanaconda/rescue.py:298 pyanaconda/rescue.py:304
msgid "Read-Only"
msgstr ""

#: pyanaconda/rescue.py:298 pyanaconda/rescue.py:300
msgid "Skip"
msgstr ""

#: pyanaconda/rescue.py:329
msgid "System to Rescue"
msgstr ""

#: pyanaconda/rescue.py:330
msgid "Which device holds the root partition of your installation?"
msgstr ""

#: pyanaconda/rescue.py:332 pyanaconda/rescue.py:336
msgid "Exit"
msgstr ""

#: pyanaconda/rescue.py:346 pyanaconda/rescue.py:422
msgid "The system will reboot automatically when you exit from the shell."
msgstr ""

#: pyanaconda/rescue.py:358
#, python-format
msgid ""
"Your system has been mounted under %(rootPath)s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"%(msg)s"
msgstr ""

#: pyanaconda/rescue.py:429
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell."
msgstr ""

#: pyanaconda/rescue.py:437
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr ""

#: pyanaconda/rescue.py:441
msgid " The system will reboot automatically when you exit from the shell."
msgstr ""

#: pyanaconda/rescue.py:445
msgid "Rescue Mode"
msgstr ""

#: pyanaconda/rescue.py:446
#, python-format
msgid "You don't have any Linux partitions. Press return to get a shell.%s"
msgstr ""

#: pyanaconda/rescue.py:458
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr ""

#: pyanaconda/storage_utils.py:50 pyanaconda/storage_utils.py:82
#: pyanaconda/ui/gui/spokes/custom.glade:31
msgid "LVM"
msgstr ""

#: pyanaconda/storage_utils.py:51 pyanaconda/storage_utils.py:83
#: pyanaconda/ui/gui/spokes/custom.glade:39
msgid "LVM Thin Provisioning"
msgstr ""

#: pyanaconda/storage_utils.py:52 pyanaconda/ui/gui/spokes/custom.glade:35
msgid "RAID"
msgstr ""

#: pyanaconda/storage_utils.py:53 pyanaconda/storage_utils.py:80
#: pyanaconda/ui/gui/spokes/custom.glade:23
msgid "Standard Partition"
msgstr ""

#: pyanaconda/storage_utils.py:54 pyanaconda/storage_utils.py:81
#: pyanaconda/ui/gui/spokes/custom.glade:27
msgid "Btrfs"
msgstr ""

#: pyanaconda/storage_utils.py:55
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:713
#: pyanaconda/ui/gui/spokes/lib/resize.glade:152
msgid "Disk"
msgstr ""

#: pyanaconda/storage_utils.py:66
msgid ""
"The 'swap' area on your computer is used by the operating\n"
"system when running low on memory."
msgstr ""

#: pyanaconda/storage_utils.py:68
msgid ""
"The 'boot' area on your computer is where files needed\n"
"to start the operating system are stored."
msgstr ""

#: pyanaconda/storage_utils.py:70
msgid ""
"The 'root' area on your computer is where core system\n"
"files and applications are stored."
msgstr ""

#: pyanaconda/storage_utils.py:72
msgid ""
"The 'home' area on your computer is where all your personal\n"
"data is stored."
msgstr ""

#: pyanaconda/storage_utils.py:74
msgid ""
"The BIOS boot partition is required to enable booting\n"
"from GPT-partitioned disks on BIOS hardware."
msgstr ""

#: pyanaconda/storage_utils.py:76
msgid ""
"The PReP boot partition is required as part of the\n"
"boot loader configuration on some PPC platforms."
msgstr ""

#: pyanaconda/storage_utils.py:184
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""

#: pyanaconda/storage_utils.py:189
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""

#: pyanaconda/storage_utils.py:203
msgid ""
"This platform requires /boot on a dedicated partition or logical volume.  If "
"you do not want a /boot volume, you must place / on a dedicated non-LVM "
"partition."
msgstr ""

#: pyanaconda/storage_utils.py:213
#, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""

#: pyanaconda/storage_utils.py:223
#, python-format
msgid ""
"Your %(mount)s partition is too small for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""

#: pyanaconda/storage_utils.py:229
#, python-format
msgid ""
"Your %(mount)s partition is too large for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""

#: pyanaconda/storage_utils.py:238
msgid "No valid boot loader target device found. See below for details."
msgstr ""

#: pyanaconda/storage_utils.py:250
msgid "You have not created a bootable partition."
msgstr ""

#: pyanaconda/storage_utils.py:272
msgid ""
"Your BIOS-based system needs a special partition to boot from a GPT disk "
"label. To continue, please create a 1MiB 'biosboot' type partition."
msgstr ""

#: pyanaconda/storage_utils.py:283
#, python-format
msgid ""
"You have not specified a swap partition.  %(requiredMem)s of memory is "
"required to continue installation without a swap partition, but you only "
"have %(installedMem)s."
msgstr ""

#: pyanaconda/storage_utils.py:290
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""

#: pyanaconda/storage_utils.py:297
msgid ""
"At least one of your swap devices does not have a UUID, which is common in "
"swap space created using older versions of mkswap. These devices will be "
"referred to by device path in /etc/fstab, which is not ideal since device "
"paths can change under a variety of circumstances. "
msgstr ""

#: pyanaconda/storage_utils.py:308
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""

#: pyanaconda/storage_utils.py:313
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr ""

#: pyanaconda/storage_utils.py:341
#, python-format
msgid ""
"Encryption requested for LUKS device %s but no encryption key specified for "
"this device."
msgstr ""

#: pyanaconda/text.py:99
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""

#: pyanaconda/text.py:106
msgid "Passphrase"
msgstr ""

#: pyanaconda/vnc.py:74
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr ""

#: pyanaconda/vnc.py:135
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr ""

#: pyanaconda/vnc.py:154
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr ""

#: pyanaconda/vnc.py:168
msgid "Connected!"
msgstr ""

#: pyanaconda/vnc.py:171
msgid "Will try to connect again in 15 seconds..."
msgstr ""

#: pyanaconda/vnc.py:178
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/vnc.py:186
msgid "Attempting to start vncconfig"
msgstr ""

#: pyanaconda/vnc.py:199
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""

#: pyanaconda/vnc.py:201
#, python-format
msgid ""
"Please manually connect your vnc client to <IP ADDRESS>:%s to begin the "
"install. Switch to the shell (Ctrl-B 2) and run 'ip addr' to find the <IP "
"ADDRESS>."
msgstr ""

#: pyanaconda/vnc.py:206
msgid "Starting VNC..."
msgstr ""

#: pyanaconda/vnc.py:243
msgid "The VNC server is now running."
msgstr ""

#: pyanaconda/vnc.py:247
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:252
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:256
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:258
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:280
msgid ""
"VNC password must be six to eight characters long.\n"
"Please enter a new one, or leave blank for no password."
msgstr ""

#: pyanaconda/packaging/__init__.py:1122
msgid "Failed to set up installation source"
msgstr ""

#: pyanaconda/packaging/__init__.py:1123
msgid "Error downloading package metadata"
msgstr ""

#: pyanaconda/packaging/__init__.py:1124
msgid "No installation source available"
msgstr ""

#: pyanaconda/packaging/livepayload.py:102
#: pyanaconda/packaging/livepayload.py:123
msgid "Installing software"
msgstr ""

#: pyanaconda/packaging/livepayload.py:230
#: pyanaconda/packaging/livepayload.py:239
#, python-format
msgid "Downloading %(url)s (%(pct)d%%)"
msgstr ""

#: pyanaconda/packaging/livepayload.py:385
msgid "Checking image checksum"
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:146
#, python-format
msgid ""
"Downloading %(total_files)s RPMs, %(downloaded)s / %(total_size)s "
"(%(percent)d%%) done."
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:613
msgid "Starting package installation process"
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:630
msgid "Downloading packages"
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:642
msgid "Preparing transaction from installation source"
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:652
#, python-format
msgid "Installing %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:163
#, python-format
msgid "Starting pull of %(branchName)s from %(source)s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:179
#, python-format
msgid "Preparing deployment of %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:199
#, python-format
msgid "Deployment starting: %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:202
#, python-format
msgid "Deployment complete: %s"
msgstr ""

#: pyanaconda/ui/helpers.py:94 pyanaconda/ui/tui/spokes/storage.py:398
msgid "Checking storage configuration..."
msgstr ""

#: pyanaconda/ui/categories/__init__.py:48
msgid "DEFAULT TITLE"
msgstr ""

#: pyanaconda/ui/categories/customization.py:32
msgid "CUSTOMIZATION"
msgstr ""

#: pyanaconda/ui/categories/localization.py:31
msgid "LOCALIZATION"
msgstr ""

#: pyanaconda/ui/categories/software.py:31
msgid "SOFTWARE"
msgstr ""

#: pyanaconda/ui/categories/system.py:32
#: pyanaconda/ui/gui/spokes/lib/accordion.py:166
msgid "SYSTEM"
msgstr ""

#: pyanaconda/ui/categories/user_settings.py:32
msgid "USER SETTINGS"
msgstr ""

#: pyanaconda/ui/lib/space.py:37
#, python-format
msgid ""
"Not enough space in file systems for the current software selection. An "
"additional %s is needed."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:58
msgid "The system needs more random data entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:67
msgid "Entropy can be increased by typing randomly on keyboard"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:68
#, python-format
msgid ""
"After %d minutes, the installation will continue regardless of the amount of "
"available entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:92
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%] "
"(%(rem)d %(min)s remaining)"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:95
#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.py:88
msgid "minute"
msgid_plural "minutes"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/lib/entropy.py:102
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%]"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:107
msgid "Enough entropy gathered, please stop typing."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:110
#, python-format
msgid "Giving up, time (%d minutes) ran out."
msgstr ""

#: pyanaconda/ui/lib/disks.py:112
#, python-format
msgid ""
"You selected disk %(selected)s, which contains devices that also use "
"unselected disk %(unselected)s. You must select or de-select these disks as "
"a set."
msgid_plural ""
"You selected disk %(selected)s, which contains devices that also use "
"unselected disks %(unselected)s. You must select or de-select these disks as "
"a set."
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/tui/hubs/summary.py:35
msgid "Installation"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:51
msgid "Starting automated install"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:85
msgid "The following mandatory spokes are not completed:"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:93
msgid ""
"  Please make your choice from above ['q' to quit | 'b' to begin "
"installation |\n"
"  'r' to refresh]: "
msgstr ""

#. TRANSLATORS: 'b' to begin installation
#: pyanaconda/ui/tui/hubs/summary.py:107
msgctxt "TUI|Spoke Navigation"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:110
#: pyanaconda/ui/tui/hubs/__init__.py:114
msgid "Please complete all spokes before continuing"
msgstr ""

#. TRANSLATORS: 'c' to continue
#: pyanaconda/ui/tui/hubs/summary.py:121
#: pyanaconda/ui/tui/hubs/__init__.py:111
#: pyanaconda/ui/tui/simpleline/base.py:457
#: pyanaconda/ui/tui/spokes/software.py:189
msgctxt "TUI|Spoke Navigation"
msgid "c"
msgstr ""

#: pyanaconda/ui/tui/hubs/__init__.py:41
msgid "Default HUB title"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:30 pyanaconda/ui/gui/main.glade:72
msgid "Error"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:51
msgid "Press enter to exit."
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:61
msgid "Question"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:84
msgid "Please respond 'yes' or 'no': "
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:87
msgid "yes"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:92
msgid "no"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:88
msgid "Do you really want to quit?"
msgstr ""

#. TRANSLATORS: 'r' to refresh
#: pyanaconda/ui/tui/simpleline/base.py:451
msgctxt "TUI|Spoke Navigation"
msgid "r"
msgstr ""

#. TRANSLATORS: 'q' to quit
#: pyanaconda/ui/tui/simpleline/base.py:463
msgctxt "TUI|Spoke Navigation"
msgid "q"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:572
msgid "Press ENTER to continue"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:622
msgid ""
"  Please make your choice from above ['q' to quit | 'c' to continue |\n"
"  'r' to refresh]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:42
msgid "VNC"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:58
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"installation or continue with a text mode installation?"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:106
msgid "q"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:122
msgid "VNC Password"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:130
msgid ""
"Please provide VNC password (must be six to eight characters long).\n"
"You will have to type it twice. Leave blank for no password"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:149
#: pyanaconda/ui/gui/spokes/network.py:1319
msgid "Password: "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:150
msgid "Password (confirm): "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:153
msgid "Passwords do not match!"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:156
msgid "The password must be at least six characters long."
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:160
msgid "The password cannot be more than eight characters long."
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:38
msgid "Language settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:65
msgid "Language is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:75
msgid "Available locales"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:78
msgid "Available languages"
msgstr ""

#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/spokes/langsupport.py:112
msgctxt "TUI|Spoke Navigation|Language Support"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:120
msgid ""
"Please select language support to install.\n"
"[b to language list, c to continue, q to quit]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:46
msgid "Network configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:107
#, python-format
msgid "Wired (%(interface_name)s) disconnected\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:112
#, python-format
msgid "Wired (%(interface_name)s) connected\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:124
#, python-format
msgid " IPv4 Address: %(addr)s Netmask: %(netmask)s Gateway: %(gateway)s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:126
#, python-format
msgid " DNS: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:133
#, python-format
msgid " IPv6 Address: %(addr)s/%(prefix)d\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:154
#, python-format
msgid "Host Name: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:166
msgid "Set host name"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:168
#, python-format
msgid "Configure device %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:187
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:241
#: pyanaconda/ui/gui/spokes/network.glade:2233
msgid "Host Name"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:205
msgid ""
"Configuration not saved: gateway or netmask missing in static configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:221
msgid "Can't apply configuration, device activation failed."
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:248
#: pyanaconda/ui/gui/spokes/network.py:1466
#: pyanaconda/ui/gui/spokes/network.py:1535
#, python-format
msgid "Host name is not valid: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:272
msgid "Device configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:276
#, python-format
msgid "IPv4 address or %s for DHCP"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:278
msgid "IPv4 netmask"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:279
msgid "IPv4 gateway"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:280
#, python-format
msgid ""
"IPv6 address or %(auto)s for automatic, %(dhcp)s for DHCP, %(ignore)s to "
"turn off"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:283
msgid "IPv6 default gateway"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:284
msgid "Nameservers (comma separated)"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:285
msgid "Connect automatically after reboot"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:286
msgid "Apply configuration in installer"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:301
#, python-format
msgid "Configuring device %s."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:32
msgid "Root password"
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:55
msgid "Password is set."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:57
msgid "Root account is disabled."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:59
msgid "Password is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:64
msgid "Please select new root password. You will have to type it twice."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:70
#: pyanaconda/ui/gui/spokes/password.glade:86
#: pyanaconda/ui/gui/spokes/user.glade:170
msgid "Password"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:55
msgid "Use All Space"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:56
msgid "Replace Existing Linux system(s)"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:57
msgid "Use Free Space"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:67
msgid "Installation Destination"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:123
#: pyanaconda/ui/gui/spokes/storage.py:374
msgid "No disks selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:126
#: pyanaconda/ui/gui/spokes/storage.py:377
msgid "Kickstart insufficient"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:133
#: pyanaconda/ui/gui/spokes/storage.py:386
msgid "Error checking storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:135
#: pyanaconda/ui/gui/spokes/storage.py:388
msgid "Warning checking storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:138
#: pyanaconda/ui/gui/spokes/storage.py:390
msgid "Automatic partitioning selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:140
#: pyanaconda/ui/gui/spokes/storage.py:392
msgid "Custom partitioning selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:176
#: pyanaconda/ui/gui/spokes/storage.py:639
msgid ""
"No disks detected.  Please shut down the computer, connect at least one "
"disk, and restart to complete installation."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:178
#: pyanaconda/ui/gui/spokes/storage.py:641
msgid "No disks selected; please select at least one disk to install to."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:193
#: pyanaconda/ui/tui/spokes/source.py:366
#: pyanaconda/ui/gui/spokes/source.py:660
#: pyanaconda/ui/gui/spokes/storage.py:590
msgid "Probing storage..."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:214
msgid "Select all"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:316
msgid ""
"The following unformatted DASDs have been detected on your system. You can "
"choose to format them now with dasdfmt or cancel to leave them unformatted. "
"Unformatted DASDs cannot be used during installation.\n"
"\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:318
msgid ""
"Warning: All storage changes made using the installer will be lost when you "
"choose to format.\n"
"\n"
"Proceed to run dasdfmt?\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:333
#: pyanaconda/ui/gui/spokes/lib/dasdfmt.py:92
#, python-format
msgid "Formatting /dev/%s. This may take a moment."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:377
msgid "Generating updated storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:382
#: pyanaconda/ui/tui/spokes/storage.py:394
#, python-format
msgid "storage configuration failed: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:441
msgid "Autopartitioning Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:467
msgid ""
"Installation requires partitioning of your hard drive. Select what space to "
"use for the install target."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:506
msgid "Partition Scheme Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:530
msgid "Select a partition scheme configuration."
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:31
msgid "Timezone settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:61
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:533
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:539
#, python-format
msgid "%s timezone"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:63
msgid "Timezone is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:71
#, python-format
msgid "Available timezones in region %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:74
msgid "Available regions"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:140
msgid ""
"Please select the timezone.\n"
"Use numbers or type names directly [b to region list, q to quit]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:33
msgid "Shell"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:48
msgid "Start shell"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:57
msgid "Exit the shell to continue"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:54
msgid "Installation source"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:57
#: pyanaconda/ui/tui/spokes/source.py:106
#: pyanaconda/ui/gui/spokes/source.py:550
#: pyanaconda/ui/gui/spokes/source.glade:757
msgid "Closest mirror"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:98
#: pyanaconda/ui/gui/spokes/source.py:542
#, python-format
msgid "NFS server %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:100
#: pyanaconda/ui/gui/spokes/source.py:544
msgid "Local media"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:103
#: pyanaconda/ui/tui/spokes/source.py:117
#: pyanaconda/ui/gui/spokes/source.py:538
msgid "Error setting up software source"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:108
#: pyanaconda/ui/tui/spokes/software.py:121
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:541
#: pyanaconda/ui/gui/spokes/software.py:224
#: pyanaconda/ui/gui/spokes/source.py:552
msgid "Nothing selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:119
#: pyanaconda/ui/tui/spokes/software.py:109
msgid "Processing..."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:135
msgid "CD/DVD"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:135
msgid "local ISO file"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:135
msgid "Network"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:139
#: pyanaconda/ui/gui/spokes/source.py:856
msgid ""
"The installation source is in use by the installer and cannot be changed."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:146
msgid "Choose an installation source type."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:243
#: pyanaconda/ui/tui/spokes/source.py:281
msgid "Specify Repo Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:247
msgid "Repo URL"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:285
msgid "<server>:/<path>"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:286
msgid "NFS mount options"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:328
msgid "Select device containing the ISO file"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:387
msgid "No mountable devices found"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:415
msgid "Select an ISO to use as install source"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:444
msgid "No *.iso files found in device root folder"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:40
msgid "Software selection"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:107
#: pyanaconda/ui/gui/spokes/software.py:208
msgid "Error checking software selection"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:111
#: pyanaconda/ui/gui/spokes/software.py:211
msgid "Installation source not set up"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:113
#: pyanaconda/ui/gui/spokes/software.py:214
msgid "Source changed - please verify"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:120
#: pyanaconda/ui/gui/spokes/software.py:222
msgid "Custom software selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:147
msgid "Installation source needs to be set up first."
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:157
msgid "Base environment"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:37
msgid "User creation"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:115 pyanaconda/ui/gui/spokes/user.py:362
msgid "No user will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:117
msgid "You must set a password"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:119 pyanaconda/ui/gui/spokes/user.py:364
#, python-format
msgid "Administrator %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:121 pyanaconda/ui/gui/spokes/user.py:366
#, python-format
msgid "User %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/warnings_spoke.py:36
msgid "Warnings"
msgstr ""

#: pyanaconda/ui/tui/spokes/warnings_spoke.py:44
#: pyanaconda/ui/gui/spokes/welcome.glade:518
msgid ""
"This hardware (or a combination thereof) is not supported by Red Hat.  For "
"more information on supported hardware, please refer to http://www.redhat."
"com/hardware."
msgstr ""

#: pyanaconda/ui/tui/spokes/progress.py:37
msgid "Progress"
msgstr ""

#: pyanaconda/ui/tui/spokes/progress.py:139
msgid "\tInstallation complete.  Press return to quit"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:49
msgid "Default spoke title"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:61
msgid "testing status..."
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:103
msgid "New value"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:129
#, python-format
msgid "%s: "
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:130
#, python-format
msgid "%s (confirm): "
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:133
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:155
msgid ""
"\n"
"Would you like to use it anyway?"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:172
msgid ""
"You have provided a password containing non-ASCII characters.\n"
"You may not be able to switch between keyboard layouts to login.\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:178
#, python-format
msgid "Enter new value for '%s' and press enter\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:186
#, python-format
msgid ""
"You have provided an invalid user name: %s\n"
"Tip: Keep your user name shorter than 32 characters and do not use spaces.\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:297
msgid "Password set."
msgstr ""

#: pyanaconda/ui/gui/__init__.py:334 widgets/src/BaseWindow.c:115
msgid "Help!"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:752 widgets/src/StandaloneWindow.c:51
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:768
msgid "_No"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:768
msgid "_Yes"
msgstr ""

#: pyanaconda/ui/gui/hubs/__init__.py:227
msgid ""
"Please complete items marked with this icon before continuing to the next "
"step."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:111
msgid "Complete!"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:210
#, python-format
msgid ""
"%s is now successfully installed on your system and ready for you to use!  "
"When you are ready, reboot your system to start using it!"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:213
msgctxt "GUI|Progress"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:221
#, python-format
msgid ""
"%s is now successfully installed, but some configuration still needs to be "
"done.\n"
"Finish it and then click the Finish configuration button please."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:226
#, python-format
msgid ""
"%s is now successfully installed and ready for you to use!\n"
"Go ahead and reboot to start using it!"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:106
#, python-format
msgid "Create a new %(container_type)s ..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:107
#, python-format
msgid "Create or select %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:109
msgid "Device reconfiguration failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:111
msgid ""
"Storage configuration reset due to unrecoverable error. <a href=\"\">Click "
"for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:142
#: pyanaconda/ui/gui/spokes/custom.glade:93
msgid "MANUAL PARTITIONING"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:368
#, python-format
msgctxt "GUI|Custom Partitioning"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/custom.py:494
#, python-format
msgid ""
"When you create mount points for your %(name)s %(version)s installation, "
"you'll be able to view their details here."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:654
#, python-format
msgid ""
"/boot/efi must be on a device of type %(oneFsType)s or %(anotherFsType)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:658
#, python-format
msgid "%(fs)s must be on a device of type %(type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:661
#: pyanaconda/ui/gui/spokes/custom.py:663
#, python-format
msgid "%s cannot be encrypted"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:665
msgid "You must create a new file system on the root device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:670
#, python-format
msgid "Device does not support RAID level selection %s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:783
msgid "Device resize request failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:856
msgid "Device reformat request failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1227
#: pyanaconda/ui/gui/spokes/custom.py:2113
#, python-format
msgid "Invalid device name: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1233
#, python-format
msgid "Specified name %s already in use."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1437
msgid "No disks assigned"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1443
#, python-format
msgctxt "GUI|Custom Partitioning|Devices"
msgid " and %d other"
msgid_plural " and %d others"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/custom.py:1501
msgid "The container is encrypted."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1537
msgid ""
"The space available to this mount point can be changed by modifying the "
"volume below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1539
msgid "This file system may not be resized."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1593
msgid ""
"Error checking storage configuration.  <a href=\"\">Click for details</a> or "
"press Done again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1595
msgid ""
"Warning checking storage configuration.  <a href=\"\">Click for details</a> "
"or press Done again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1701
#, python-format
msgid "Added new %(type)s to existing container %(name)s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1712
msgid "Failed to add new device. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1717
msgid "Invalid partition size set. Use a valid integer."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1824
msgid "Device removal request failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2054
#, python-format
msgid "Volume Group name %s is already in use. Not saving changes."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2084
#, python-format
msgid "(%s free)"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2231
#, python-format
msgid ""
"This Software RAID array is missing %(missingMembers)d of %(totalMembers)d "
"member partitions. You can remove it or select a different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2237
#, python-format
msgid ""
"This LVM Volume Group is missing %(missingPVs)d of %(totalPVs)d physical "
"volumes. You can remove it or select a different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2320
msgid "No disks selected."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2325
msgid "Not enough free space on selected disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2330
#: pyanaconda/ui/gui/spokes/custom.py:2355
msgid "Automatic partitioning failed. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2579
msgid ""
"Continuing with this action will reset all your partitioning selections to "
"their current on-disk state."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2587
msgid "_Reset selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2587
msgid "_Preserve current selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2662
msgid ""
"Failed to unlock encrypted block device. <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:401
msgctxt "GUI|Spoke"
msgid "_TIME & DATE"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:524
msgid "Restoring hardware time..."
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:535
msgid "Invalid timezone"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:711
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:725
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:911
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:751
msgid "PM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:713
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:722
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:910
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:913
msgid "AM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1061
msgid "You need to set up networking first if you want to use NTP"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1065
msgid "You have no working NTP server configured"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.py:482
msgctxt "GUI|Spoke"
msgid "_INSTALLATION DESTINATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.py:583
#, python-format
msgctxt "GUI|Installation Destination|Filter"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/keyboard.py:48
#, python-format
msgid "%s to switch layouts."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:272
msgctxt "GUI|Spoke"
msgid "_KEYBOARD"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:359
msgid "Testing layouts configuration not available."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:367
msgid "Getting list of layouts..."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:422
msgid "Layout switching not configured."
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.py:54
msgctxt "GUI|Spoke"
msgid "_LANGUAGE SUPPORT"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:87
msgid "Status unknown (missing)"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:90
msgid "Status unknown"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:92
msgid "Unmanaged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:95
#: pyanaconda/ui/gui/spokes/network.py:99
msgid "Unavailable"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:97
msgid "Firmware missing"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:103
msgid "Cable unplugged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:105
msgid "Disconnected"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:110
msgid "Connecting"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:112
msgid "Authentication required"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:114
msgid "Connected"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:116
msgid "Disconnecting"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:118
msgid "Connection failed"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:303
msgid "Ethernet"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:304
msgid "Wireless"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:305
#: pyanaconda/ui/gui/spokes/network.glade:15
msgid "Bond"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:306
#: pyanaconda/ui/gui/spokes/network.glade:27
msgid "VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:307
#: pyanaconda/ui/gui/spokes/network.glade:23
msgid "Team"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:308
#: pyanaconda/ui/gui/spokes/network.glade:19
msgid "Bridge"
msgstr ""

#. TRANSLATORS: ethernet cable is unplugged
#: pyanaconda/ui/gui/spokes/network.py:729
msgid "unplugged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:819
#: pyanaconda/ui/gui/spokes/network.glade:540
#: pyanaconda/ui/gui/spokes/network.glade:841
msgid "IPv4 Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:820
#: pyanaconda/ui/gui/spokes/network.glade:516
#: pyanaconda/ui/gui/spokes/network.glade:853
#: pyanaconda/ui/gui/spokes/network.glade:1420
msgid "IPv6 Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:822
#: pyanaconda/ui/gui/spokes/network.py:824
#: pyanaconda/ui/gui/spokes/network.glade:1408
msgid "IP Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:914
#, python-format
msgid "%d Mb/s"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1134
msgid "WEP"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1137
msgid "WPA"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1140
msgid "WPA2"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1144
msgid "Enterprise"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1149 widgets/src/SpokeSelector.c:59
#: pyanaconda/ui/gui/spokes/filter.glade:145
#: pyanaconda/ui/gui/spokes/filter.glade:559
#: pyanaconda/ui/gui/spokes/filter.glade:893
#: pyanaconda/ui/gui/spokes/filter.glade:1213
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:531
msgid "None"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1281
#: pyanaconda/ui/gui/spokes/network.glade:2435
msgid "Authentication required by wireless network"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1282
#, python-format
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network '%(network_id)s'."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1296
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1304
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Key:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1314
msgid "User name: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1325
msgid "Identity: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1330
msgid "Private key password: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1386
msgctxt "GUI|Spoke"
msgid "_NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:52
msgctxt "GUI|Spoke"
msgid "_ROOT PASSWORD"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:99
#: pyanaconda/ui/gui/spokes/password.py:100
msgid "The password is set."
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:127
msgid "Root password is set"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:129
msgid "Root account is disabled"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:131
msgid "Root password is not set"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:52
msgctxt "GUI|Spoke"
msgid "_SOFTWARE SELECTION"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:98
#: pyanaconda/ui/gui/spokes/source.py:55
msgid "Downloading package metadata..."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:101
msgid "Downloading group metadata..."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:143
#: pyanaconda/ui/gui/spokes/source.py:532
msgid "Checking software dependencies..."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:148
msgid "Error checking software dependencies"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:374
msgid ""
"Error checking software dependencies.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:478
msgid ""
"The software marked for installation has the following errors.  This is "
"likely caused by an error with your installation source.  You can quit the "
"installer, change your software source, or change your software selections."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:483
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:484
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Modify Software Source"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:485
msgctxt "GUI|Software Selection|Error Dialog"
msgid "Modify _Selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:54
msgid "Setting up installation source..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:84
msgid "Invalid proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:89
#, python-format
msgid "Invalid proxy protocol: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:95
msgid "Extra characters in proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:99
msgid "Proxy authentication data duplicated"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:232
msgid "This media is good to install from."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:234
msgid "This media is not good to install from."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:357
msgctxt "GUI|Spoke"
msgid "_INSTALLATION SOURCE"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:472
#: pyanaconda/ui/gui/spokes/source.py:676
msgid "Failed to set up installation source; check the repo url"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:536
msgid "Error setting up base repository"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:547
msgid "Error setting up ISO file"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:678
msgid ""
"Failed to set up installation source; check the repo url and proxy settings"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:707
#, python-format
msgid "Device: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:708
#, python-format
msgid "Label: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:868
msgid "You need to configure the network to use a network installation source."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:920
msgid "URL is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:924
msgid "Invalid URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:941
msgid "Protocol in URL does not match selected protocol"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:944
msgid "NFS server is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:951
msgid "Invalid host name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:954
msgid "Remote directory is required"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:975
msgid "Duplicate repository names."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:982
msgid "Empty repository name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:985
msgid "Invalid repository name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:991
msgid "Repository name conflicts with internal repository name."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:112
msgid "Please wait... software metadata still loading."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:115
#, python-format
msgid ""
"Your current <b>%(product)s</b> software selection requires <b>%(total)s</b> "
"of available space, including <b>%(software)s</b> for software and <b>"
"%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:124
#, python-format
msgid ""
"Your current <a href=\"\" title=\"%(tooltip)s\"><b>%(product)s</b> software "
"selection</a> requires <b>%(total)s</b> of available space, including <b>"
"%(software)s</b> for software and <b>%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:174
#, python-format
msgid "%s The disks you've selected have the following amounts of free space:"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:184
#, python-format
msgid ""
"<b>You don't have enough space available to install %s</b>.  You can shrink "
"or remove existing partitions via our guided reclaim space tool, or you can "
"adjust your partitions on your own in the custom partitioning interface."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:208
#, python-format
msgid ""
"  You don't have enough space available to install <b>%(product)s</b>, even "
"if you used all of the free space available on the selected disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:220
#, python-format
msgid ""
"<b>You don't have enough space available to install %(productName)s</b>, "
"even if you used all of the free space available on the selected disks.  You "
"could add more disks for additional space, modify your software selection to "
"install a smaller version of <b>%(productName)s</b>, or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:241
msgctxt "GUI|Spoke"
msgid "INSTALLATION _DESTINATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:325
msgid "Saving storage configuration..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:331
#: pyanaconda/ui/gui/spokes/storage.py:343
msgid "Failed to save storage configuration..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:379
#: pyanaconda/ui/gui/spokes/storage.py:672
msgid "Formatting DASDs"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:521
msgid ""
"Error checking storage configuration.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:523
msgid ""
"Warning checking storage configuration.  <a href=\"\">Click for details.</a>"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:564
#, python-format
msgid ""
"FCP device %(hba_id)s\n"
"WWPN %(wwpn)s\n"
"LUN %(lun)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:574
#, python-format
msgid "%s free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:621
#, python-format
msgid "%(count)d disk selected; %(capacity)s capacity; %(free)s free"
msgid_plural "%(count)d disks selected; %(capacity)s capacity; %(free)s free"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/storage.py:705
msgid ""
"You have chosen to skip boot loader installation.  Your system may not be "
"bootable."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:762
msgid "There was a problem with your disk selection. Click here for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:941
msgid "You'll be able to make space available during custom partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:957
msgid ""
"The following errors were encountered when checking your storage "
"configuration.  You can modify your storage layout or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:962
msgctxt "GUI|Storage|Error Dialog"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:963
msgctxt "GUI|Storage|Error Dialog"
msgid "_Modify Storage Layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:977
msgid ""
"The following warnings were encountered when checking your storage "
"configuration.  These are not fatal, but you may wish to make changes to "
"your storage layout."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:981
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:72
#, python-format
msgid "Invalid group name: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:215
msgctxt "GUI|Spoke"
msgid "_USER CREATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:293 pyanaconda/ui/gui/spokes/user.py:294
msgid "The password was set by kickstart."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:329
msgid "Invalid user name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:331
msgid "Full name cannot contain colon characters"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.py:258
#, python-format
msgid "WELCOME TO %(name)s %(version)s."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.py:270
#, python-format
msgid ""
"The following nodes were discovered using the iSCSI initiator <b>"
"%(initiatorName)s</b> using the target IP address <b>%(targetAddress)s</b>.  "
"Please select which nodes you wish to log into:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:157
msgid "DATA"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:282
#, python-format
msgid ""
"You haven't created any mount points for your %(product)s %(version)s "
"installation yet.  You can:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:291
msgid "_Click here to create them automatically."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:316
msgid "Create new mount points by clicking the '+' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:324
msgid ""
"Or, assign new mount points to existing partitions after selecting them "
"below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:329
msgid "_New mount points will use the following partitioning scheme:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:131
#, python-format
msgid ""
"<b>%(count)d disk; %(size)s capacity; %(free)s free space</b> (unpartitioned "
"and in file systems)"
msgid_plural ""
"<b>%(count)d disks; %(size)s capacity; %(free)s free space</b> "
"(unpartitioned and in file systems)"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:190
msgctxt "GUI|Selected Disks Dialog"
msgid "_Do not install boot loader"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:193
#: pyanaconda/ui/gui/spokes/lib/cart.glade:183
msgctxt "GUI|Selected Disks Dialog"
msgid "_Set as Boot Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:63
#, python-format
msgid ""
"The RAID level you have selected (%(level)s) requires more disks (%(min)d) "
"than you currently have selected (%(count)d)."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:67
#, python-format
msgid "CONFIGURE %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:68
#, python-format
msgid ""
"Please create a name for this %(container_type)s and select at least one "
"disk below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:73
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:74
msgid "Volume Group"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:73
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:74
msgid "_Volume Group:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:75
msgid "Volume"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:75
msgid "_Volume:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:135
msgid "Cannot relabel already existing file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:140
msgid "Cannot set label on file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:142
msgid "Unacceptable label format for file system."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:152
msgid "That mount point is already in use. Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:154
msgid "Please enter a valid mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:156
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:168
msgid "That mount point is invalid. Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:309
msgid "container"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:388
#, python-format
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "Delete _all other file systems in the %s root as well."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:398
#, python-format
msgid "Are you sure you want to delete all of the data on %s?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:400
#, python-format
msgid ""
"Are you sure you want to delete all of the data on %s, including snapshots "
"and/or subvolumes?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:684
msgid "Invalid container name"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:49
msgctxt "GUI|Detailed Error Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:59
msgctxt "GUI|Detailed Error Dialog"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:35
#, python-format
msgid "You have provided a weak passphrase: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:36
msgid "Passphrases do not match."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:165
msgid "Passphrase contains non-ASCII characters"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:53
msgid "Preserve"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:54
msgid "Shrink"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:55
msgid "Delete"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:114
msgid "This device contains the installation source."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:160
#, python-format
msgid "%(freeSize)s of %(devSize)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:167
msgid "Not resizeable"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:191
msgid "Free space"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:213
msgid ""
"You can remove existing file systems you no longer need to free up space for "
"this installation.  Removing a file system will permanently delete all of "
"the data it contains."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:219
msgid ""
"There is also free space available in pre-existing file systems.  While it's "
"risky and we recommend you back up your data first, you can recover that "
"free disk space and make it available for this installation below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:229
#, python-format
msgid "<b>%(count)s disk; %(size)s reclaimable space</b> (in file systems)"
msgid_plural ""
"<b>%(count)s disks; %(size)s reclaimable space</b> (in file systems)"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:236
#, python-format
msgid "Total selected space to reclaim: <b>%s</b>"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:451
#: pyanaconda/ui/gui/spokes/lib/resize.py:456
#: pyanaconda/ui/gui/spokes/lib/resize.glade:269
msgctxt "GUI|Reclaim Dialog"
msgid "Delete _all"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:453
msgctxt "GUI|Reclaim Dialog"
msgid "Preserve _all"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.py:54
#, python-format
msgid "%(description)s (%(deviceName)s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.py:58
#, python-format
msgid "%(deviceName)s on %(container)s"
msgstr ""

#: widgets/src/BaseWindow.c:111
msgid "DISTRIBUTION INSTALLATION"
msgstr ""

#: widgets/src/BaseWindow.c:112
msgid "SPOKE NAME"
msgstr ""

#: widgets/src/BaseWindow.c:113
msgid "PRE-RELEASE / TESTING"
msgstr ""

#: widgets/src/DiskOverview.c:56
msgid "New Device"
msgstr ""

#: widgets/src/DiskOverview.c:58 widgets/src/DiskOverview.c:59
msgid "0 MB"
msgstr ""

#: widgets/src/MountpointSelector.c:55
msgid "0 GB"
msgstr ""

#: widgets/src/SpokeSelector.c:60
msgid "New Selector"
msgstr ""

#: widgets/src/SpokeWindow.c:48
msgid "_Done"
msgstr ""

#: widgets/src/StandaloneWindow.c:52
msgid "_Continue"
msgstr ""

#: widgets/src/LayoutIndicator.c:34
#, c-format
msgid "Current layout: '%s'. Click to switch to the next layout."
msgstr ""

#: widgets/src/LayoutIndicator.c:35
#, c-format
msgid "Current layout: '%s'. Add more layouts to enable switching."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:36
msgctxt "GUI|Advanced User"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:50
msgctxt "GUI|Advanced User"
msgid "_Save Changes"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:81
msgid "ADVANCED USER CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:103
msgctxt "GUI|Advanced User"
msgid "Home _directory:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:138
msgid "User and Group IDs"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:157
msgctxt "GUI|Advanced User"
msgid "Specify a _user ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:173
msgctxt "GUI|Advanced User"
msgid "Specify a _group ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:231
msgid "Group Membership"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:259
msgctxt "GUI|Advanced User"
msgid "_Add user to the following groups:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:290
msgid "Example:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:307
msgid "wheel, my-team (1245), project-x (29935)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:341
msgid "Tip:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:360
msgid ""
"You may input a comma-separated list of group names and group IDs here. "
"Groups that do not already exist will be created; specify their GID in "
"parentheses. "
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:110
msgctxt "GUI|Date and Time|NTP"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:124
msgctxt "GUI|Date and Time|NTP"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:149
msgid "Add and mark for usage NTP servers"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:171
msgid "New NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:190
msgid "Add NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:210
msgid "This URL refers to a pool of NTP servers"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:256
msgid "Pool"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:269
msgid "Working"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:277
msgid "Use"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:305
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:519
msgid "Configure NTP"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:326
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:934
msgid "TIME & DATE"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:376
msgctxt "GUI|Date and Time"
msgid "_Region:"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:406
msgid "Region"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:423
msgctxt "GUI|Date and Time"
msgid "_City:"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:453
msgid "City"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:481
msgctxt "GUI|Date and Time"
msgid "_Network Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:499
msgid "Use Network Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:586
msgid "Hours"
msgstr ""

#. TRANSLATORS: This is the separator between hours and minutes, like in HH:MM
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:599
msgid ":"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:619
msgid "Minutes"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:637
msgid "Hour Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:655
msgid "Hour Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:673
msgid "Minutes Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:691
msgid "Minutes Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:735
msgid "AM/PM Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:771
msgid "AM/PM Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:822
msgctxt "GUI|Date and Time"
msgid "24-_hour"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:840
msgctxt "GUI|Date and Time"
msgid "_AM/PM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:899
msgid "Set Date & Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:60
#: pyanaconda/ui/gui/spokes/storage.glade:444
#: pyanaconda/ui/gui/spokes/storage.glade:1024
msgid "INSTALLATION DESTINATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:129
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:146
msgid "Port / Target / LUN #"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:147
msgid "Target WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:186
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Port:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:212
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Target:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:241
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:283
msgctxt "GUI|Installation Destination|Filter|Search|WWID"
msgid "_WWID:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:333
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search Res_ults:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:379
#: pyanaconda/ui/gui/spokes/filter.glade:1437
#: pyanaconda/ui/gui/spokes/keyboard.glade:520
#: pyanaconda/ui/gui/spokes/source.glade:987
#: pyanaconda/ui/gui/spokes/lib/cart.glade:133
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:235
#: pyanaconda/ui/gui/spokes/lib/resize.glade:163
msgid "Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:393
#: pyanaconda/ui/gui/spokes/filter.glade:562
#: pyanaconda/ui/gui/spokes/filter.glade:765
msgid "WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:407
#: pyanaconda/ui/gui/spokes/filter.glade:779
#: pyanaconda/ui/gui/spokes/filter.glade:1113
#: pyanaconda/ui/gui/spokes/filter.glade:1465
#: pyanaconda/ui/gui/spokes/lib/cart.glade:145
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:248
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:726
msgid "Capacity"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:421
#: pyanaconda/ui/gui/spokes/filter.glade:561
#: pyanaconda/ui/gui/spokes/filter.glade:807
#: pyanaconda/ui/gui/spokes/filter.glade:895
#: pyanaconda/ui/gui/spokes/filter.glade:1141
msgid "Interconnect"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:435
msgid "Model"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:449
#: pyanaconda/ui/gui/spokes/filter.glade:1216
#: pyanaconda/ui/gui/spokes/filter.glade:1493
msgid "LUN"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:463
#: pyanaconda/ui/gui/spokes/filter.glade:1507
msgid "Port"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:477
msgid "Target"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:491
#: pyanaconda/ui/gui/spokes/filter.glade:560
#: pyanaconda/ui/gui/spokes/filter.glade:793
#: pyanaconda/ui/gui/spokes/filter.glade:894
#: pyanaconda/ui/gui/spokes/filter.glade:1127
msgid "Vendor"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:518
msgctxt "GUI|Installation Destination|Filter"
msgid "Searc_h"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:543
msgctxt "GUI|Installation Destination|Filter|Multipath"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:600
msgctxt "GUI|Installation Destination|Filter|Multipath|Vendor"
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:641
msgctxt "GUI|Installation Destination|Filter|Multipath|Interconnect"
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:682
msgctxt "GUI|Installation Destination|Filter|Multipath|WWID"
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:821
msgid "Paths"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:851
msgctxt "GUI|Installation Destination|Filter"
msgid "_Multipath Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:877
msgctxt "GUI|Installation Destination|Filter|Other"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:896
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:752
msgid "ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:934
msgctxt "GUI|Installation Destination|Filter|Other|Vendor"
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:975
msgctxt "GUI|Installation Destination|Filter|Other|Interconnect"
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1016
msgctxt "GUI|Installation Destination|Filter|Other|ID"
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1099
msgid "Identifier"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1171
msgctxt "GUI|Installation Destination|Filter"
msgid "_Other SAN Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1197
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "Filter B_y:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1214
#: pyanaconda/ui/gui/spokes/filter.glade:1423
msgid "CCW"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1215
#: pyanaconda/ui/gui/spokes/filter.glade:1479
msgid "WWPN"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1253
msgctxt "GUI|Installation Destination|Filter|zSeries|CCW"
msgid "_CCW:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1296
msgctxt "GUI|Installation Destination|Filter|zSeries|WWPN"
msgid "_WWPN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1340
msgctxt "GUI|Installation Destination|Filter|zSeries|LUN"
msgid "_LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1451
#: pyanaconda/ui/gui/spokes/lib/summary.glade:155
msgid "Type"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1537
msgctxt "GUI|Installation Destination|Filter"
msgid "_zSeries Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1560
msgctxt "GUI|Installation Destination|Filter"
msgid "_Add zFCP LUN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1575
msgctxt "GUI|Installation Destination|Filter"
msgid "Add EC_KD DASD..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1590
msgctxt "GUI|Installation Destination|Filter"
msgid "Add _iSCSI Target..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1605
msgctxt "GUI|Installation Destination|Filter"
msgid "Add FCo_E SAN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1620
msgctxt "GUI|Installation Destination|Filter"
msgid "Refresh _List"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:35
msgid "Add device"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:49
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:63
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Add"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:92
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Select the type of device you wish to add"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:165
#: pyanaconda/ui/gui/spokes/network.glade:2270
msgid "NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:204
msgid ""
"Please use the live desktop environment's tools for customizing your network "
"configuration.  You can set the host name here."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:418
msgid "Slaves"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:442
#: pyanaconda/ui/gui/spokes/network.glade:878
#: pyanaconda/ui/gui/spokes/network.glade:1445
msgid "DNS"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:468
#: pyanaconda/ui/gui/spokes/network.glade:865
#: pyanaconda/ui/gui/spokes/network.glade:1432
msgid "Default Route"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:492
msgid "Subnet Mask"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:564
#: pyanaconda/ui/gui/spokes/network.glade:829
#: pyanaconda/ui/gui/spokes/network.glade:1276
msgid "Speed"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:588
#: pyanaconda/ui/gui/spokes/network.glade:817
msgid "Hardware Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:612
msgid "Parent"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:624
msgid "VLAN ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:699
msgctxt "GUI|Network|Wired"
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:703
msgid "To apply the configuration immediately turn the device off and on."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:990
msgid "Security"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1002
msgctxt "GUI|Network|Wireless"
msgid "_Network Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1016
msgid "Network Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1028
msgid "Security Key"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1111
msgctxt "GUI|Network|Wireless"
msgid "_Use as Hotspot..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1128
msgctxt "GUI|Network|Wireless"
msgid "_Stop Hotspot..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1145
msgctxt "GUI|Network|Wireless"
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1264
msgid "IMEI"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1288
msgid "Provider"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1470
msgctxt "GUI|Network|Mobile Broadband"
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1581
msgid "VPN Type"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1593
msgid "Gateway"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1605
msgid "Group Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1617
msgid "Group Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1629
#: pyanaconda/ui/gui/spokes/user.glade:116
msgid "User Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1736
msgctxt "GUI|Network|VPN"
msgid "_Configure..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1859
msgctxt "GUI|Network|Proxy"
msgid "_Method"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1884
msgctxt "GUI|Network|Proxy"
msgid "_Configuration URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1909
msgctxt "GUI|Network|Proxy"
msgid "HTTP _Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1923
msgctxt "GUI|Network|Proxy"
msgid "H_TTPS Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1937
msgctxt "GUI|Network|Proxy"
msgid "_FTP Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1951
msgctxt "GUI|Network|Proxy"
msgid "_Socks Host"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2140
msgid "Network Config Box"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2156
msgctxt "GUI|Network"
msgid "Unloc_k"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2177
msgctxt "GUI|Network"
msgid "_Airplane Mode"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2196
msgid "More Network Config Box"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2216
msgctxt "GUI|Network"
msgid "_Host Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2276
msgid "NETWORK CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2321
msgid ""
"We'll need network access to fetch information about your location and to "
"make software\n"
"updates available for you."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2351
msgid "Authentication"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2365
msgctxt "GUI|Network|Authentication Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2380
msgctxt "GUI|Network|Authentication Dialog"
msgid "C_onnect"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2454
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade:10
msgid "SOFTWARE SELECTION"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade:66
msgid "Base Environment"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade:84
msgid "Add-Ons for Selected Environment"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:27
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Cancel & add more disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:43
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Reclaim space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:78
#: pyanaconda/ui/gui/spokes/storage.glade:293
msgid "INSTALLATION OPTIONS"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:95
#: pyanaconda/ui/gui/spokes/storage.glade:309
msgid ""
"Here we'll describe how much space is needed for the current software "
"selection."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:121
#: pyanaconda/ui/gui/spokes/storage.glade:335
msgid "disk free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:137
#: pyanaconda/ui/gui/spokes/storage.glade:350
msgid "Free space available for use."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:153
#: pyanaconda/ui/gui/spokes/storage.glade:366
msgid "fs free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:169
#: pyanaconda/ui/gui/spokes/storage.glade:382
msgid "Free space unavailable but reclaimable from existing partitions."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:192
#: pyanaconda/ui/gui/spokes/storage.glade:405
msgid "Here we'll describe what your options are."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:219
msgid "Need Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:242
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Quit installer"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:258
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:432
msgid "No Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:507
msgid "Device Selection"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:526
msgid ""
"Select the device(s) you'd like to install to.  They will be left untouched "
"until you click on the main menu's \"Begin Installation\" button."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:544
msgid "Local Standard Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:608
#: pyanaconda/ui/gui/spokes/storage.glade:748
msgid "Disks left unselected here will not be touched."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:627
msgid "Specialized & Network Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:697
msgctxt "GUI|Storage"
msgid "_Add a disk..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:714
msgid "Add Specialized Disk"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:767
msgid "Other Storage Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:793
msgid "Partitioning"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:810
msgid "Encryption"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:823
msgid "A_utomatically configure partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:839
msgid "_I will configure partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:871
msgid "_Encrypt my data."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:886
msgid "You'll set a passphrase next."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:907
msgid "I would like to _make additional space available."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:923
msgid "Storage Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:955
msgid "summary"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:988
msgctxt "GUI|Storage"
msgid "_Full disk summary and boot loader..."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:15
#: pyanaconda/ui/gui/spokes/keyboard.glade:388
msgid "KEYBOARD LAYOUT"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:57
msgid ""
"Which keyboard layouts would you like to use on this system?  You may move "
"any layout to the top of the list to select it as the default."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:96
msgid ""
"Changes here will only apply to the installed system. Use the desktop's tool "
"to configure the keyboard for the installation process."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:153
msgid "name"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:167
msgid "Selected Layouts"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:190
msgctxt "GUI|Keyboard Layout"
msgid "_Add layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:205
msgctxt "GUI|Keyboard Layout"
msgid "_Remove layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:220
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _up"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:235
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _down"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:251
msgctxt "GUI|Keyboard Layout"
msgid "_Preview layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:291
msgctxt "GUI|Keyboard Layout"
msgid "_Test the layout configuration below:"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:331
msgid "Alt + Shift to switch layouts."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:346
msgctxt "GUI|Keyboard Layout"
msgid "_Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:421
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:435
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Add"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:467
msgid "ADD A KEYBOARD LAYOUT"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:484
msgid "You may add a keyboard layout by selecting it below:"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:533
msgid "Available Layouts"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:574
msgid "Add Layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:606
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:620
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:652
msgid "LAYOUT SWITCHING OPTIONS"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:669
msgid ""
"Which combination(s) would you prefer for switching between keyboard layouts?"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:705
msgid "use"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:718
msgid "description"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:752
msgid "Layout Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:33
msgid "LANGUAGE SUPPORT"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:79
msgid "Select additional language support to be installed:"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:138
#: pyanaconda/ui/gui/spokes/langsupport.glade:241
#: pyanaconda/ui/gui/spokes/welcome.glade:273
#: pyanaconda/ui/gui/spokes/welcome.glade:372
msgid "nativeName"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:155
#: pyanaconda/ui/gui/spokes/welcome.glade:290
msgid "englishName"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:173
#: pyanaconda/ui/gui/spokes/welcome.glade:308
msgid "selected"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:193
#: pyanaconda/ui/gui/spokes/welcome.glade:336
msgid "Type here to search."
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:227
msgid "checked"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:8
#: pyanaconda/ui/gui/spokes/password.glade:203
msgid "ROOT PASSWORD"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:54
msgctxt "GUI|Password"
msgid "_Root Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:68
msgctxt "GUI|Password"
msgid "_Confirm:"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:104
#: pyanaconda/ui/gui/spokes/user.glade:187
msgid "Confirm Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:150
#: pyanaconda/ui/gui/spokes/user.glade:251
msgid "empty password"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:171
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:29
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:43
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Open"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:95
msgctxt "GUI|Software Source|Media Check Dialog"
msgid "_Done"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:128
msgid "MEDIA VERIFICATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:155
msgid "Verifying media, please wait..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:224
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:239
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:267
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Enable HTTP Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:300
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Proxy URL:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:317
msgid "<span size=\"small\"><b>Example:</b> squid.mysite.org:3128</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:348
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Use Authentication"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:375
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "User _name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:392
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:471
msgid "INSTALLATION SOURCE"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:530
msgid "Which installation source would you like to use?"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:544
msgctxt "GUI|Software Source"
msgid "_Auto-detected installation media:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:571
msgid "Device:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:583
msgid "Label:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:592
#: pyanaconda/ui/gui/spokes/source.glade:692
msgctxt "GUI|Software Source"
msgid "_Verify"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:616
msgctxt "GUI|Software Source"
msgid "_ISO file:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:644
msgctxt "GUI|Software Source"
msgid "D_evice:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:675
msgctxt "GUI|Software Source"
msgid "_Choose an ISO"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:716
msgctxt "GUI|Software Source"
msgid "_On the network:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:779
msgctxt "GUI|Software Source"
msgid "_Proxy setup..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:794
msgctxt "GUI|Software Source"
msgid "This URL refers to a _mirror list."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:832
msgctxt "GUI|Software Source"
msgid "N_FS mount options:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:847
#: pyanaconda/ui/gui/spokes/source.glade:848
msgid "This field is optional."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:881
msgid "Updates"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:908
msgctxt "GUI|Software Source"
msgid ""
"Don't install the latest available software _updates. Install the default "
"versions provided by the installation source above."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:933
msgid "Additional repositories"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:974
msgid "Enabled"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1019
#: pyanaconda/ui/gui/spokes/source.glade:1020
msgid "Add a new repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1021
msgctxt "GUI|Software Source"
msgid "A_dd"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1038
#: pyanaconda/ui/gui/spokes/source.glade:1039
msgid "Remove the selected repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1040
msgctxt "GUI|Software Source"
msgid "_Remove"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1057
#: pyanaconda/ui/gui/spokes/source.glade:1058
msgid "Revert to the previous list of repositories."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1059
msgctxt "GUI|Software Source"
msgid "Rese_t"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1089
#: pyanaconda/ui/gui/spokes/source.glade:1090
msgid "URL for the repository, without protocol."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1102
#: pyanaconda/ui/gui/spokes/source.glade:1103
msgid "URL of proxy in the form of protocol://host:[port]"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1115
#: pyanaconda/ui/gui/spokes/source.glade:1116
msgid "Optional proxy user name."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1128
msgid "Optional proxy password."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1140
msgctxt "GUI|Software Source"
msgid "This URL refers to a mirror _list."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1160
msgctxt "GUI|Software Source"
msgid "_Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1176
#: pyanaconda/ui/gui/spokes/source.glade:1177
msgid "Protocol for the repository URL."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1198
msgctxt "GUI|Software Source"
msgid "Pro_xy URL:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1215
msgctxt "GUI|Software Source"
msgid "U_ser name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1232
msgctxt "GUI|Software Source"
msgid "Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1248
#: pyanaconda/ui/gui/spokes/source.glade:1249
msgid "Name of the repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:23
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "_Get me out of here!"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:37
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "_I accept my fate."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:68
msgid "This is unstable, pre-release software."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:97
msgid ""
"You wake up inside an OS installer in Timbuktu, and it's six months in the "
"future.  But, there are bugs.  Bugs everywhere.  Bugs you must live with.  "
"This OS of the future isn't a stable OS you can rely on.  It's for testing "
"purposes only."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:126
msgid "Beta Warn"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:225
msgid "What language would you like to use during the installation process?"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:316
msgid "Languages"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:388
msgid "Locales"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:422
msgid "WELCOME"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:443
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:457
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Continue"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:488
msgid "Unsupported Hardware Detected"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:546
msgid "Unsupported Hardware"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:10
#: pyanaconda/ui/gui/spokes/user.glade:346
msgid "CREATE USER"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:62
msgctxt "GUI|User"
msgid "_Full name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:80
msgctxt "GUI|User"
msgid "_User name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:100
msgid "Full Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:131
msgctxt "GUI|User"
msgid "_Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:149
msgctxt "GUI|User"
msgid "_Confirm password"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:201
msgid ""
"<b>Tip:</b> Keep your user name shorter than 32 characters and do not use "
"spaces."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:211
msgctxt "GUI|User"
msgid "_Require a password to use this account"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:270
msgctxt "GUI|User"
msgid "_Make this user administrator"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:289
msgctxt "GUI|User"
msgid "_Advanced..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:62
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:539
msgid "RAID0 <span foreground=\"grey\">(Performance)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:66
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:543
msgid "RAID1 <span foreground=\"grey\">(Redundancy)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:70
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:547
msgid "RAID4 <span foreground=\"grey\">(Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:74
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:551
msgid "RAID5 <span foreground=\"grey\">(Distributed Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:78
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:555
msgid "RAID6 <span foreground=\"grey\">(Redundant Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:82
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:559
msgid "RAID10 <span foreground=\"grey\">(Performance, Redundancy)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:181
msgid "Add a new mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:182
msgctxt "GUI|Custom Partitioning"
msgid "Add"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:197
msgid "Remove the selected mount point(s)."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:198
msgctxt "GUI|Custom Partitioning"
msgid "Remove"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:213
msgid "Reload storage configuration from disk."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:214
msgctxt "GUI|Custom Partitioning"
msgid "Refresh"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:279
#: pyanaconda/ui/gui/spokes/custom.glade:1012
#: pyanaconda/ui/gui/spokes/custom.glade:1156
#: pyanaconda/ui/gui/spokes/custom.glade:1243
msgid "Selected Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:297
#: pyanaconda/ui/gui/spokes/custom.glade:1030
#: pyanaconda/ui/gui/spokes/custom.glade:1174
#: pyanaconda/ui/gui/spokes/custom.glade:1261
msgid "Device description"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:358
msgid "Mount _Point:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:405
msgid "_Desired Capacity:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:456
msgid "_Label:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:503
msgid "Device(s):"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:532
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Modify..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:564
msgid "_Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:619
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Device _Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:663
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Encrypt"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:704
msgctxt "GUI|Custom Partitioning|Configure"
msgid "File S_ystem:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:739
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Ref_ormat"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:793
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Volume Group:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:847
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Modify..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:879
msgctxt "GUI|Custom Partitioning|Configure"
msgid "RA_ID Level:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:944
msgid "_Update Settings"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:967
msgid ""
"Note:  The settings you make on this screen will not be applied until you "
"click on the main menu's 'Begin Installation' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1064
msgid ""
"This device is encrypted and cannot be read without a valid passphrase. You "
"may unlock it below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1085
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Passphrase:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1113
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Unlock"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1208
msgid ""
"This device cannot be edited directly. You can remove it or select a "
"different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1356
msgid "AVAILABLE SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1409
msgid "TOTAL SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1449
msgctxt "GUI|Custom Partitioning"
msgid "_Reset All"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:23
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:56
msgid ""
"Please select the network interface which is connected to your FCoE switch."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:70
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_NIC:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:92
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use _DCB"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:108
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use auto _VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:144
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Add FCoE Disk(s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:187
msgid "Error adding FCoE SAN."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:215
msgid ""
"The following error occurred while attempting to add an FCoE SAN.  Please "
"check your configuration and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:40
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:54
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:86
msgid "ADD iSCSI STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:115
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:142
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Target IP Address:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:169
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "iSCSI _Initiator Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:185
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:751
msgid "No credentials (discovery authentication disabled)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:186
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:752
msgid "CHAP pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:187
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:753
msgid "CHAP pair and a reverse pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:202
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Discovery Authentication Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:217
msgid ""
"<span size=\"small\"><b>Example:</b> iqn.2012-09.com.example:diskarrays-sn-"
"a8675309</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:262
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:276
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:382
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:396
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:410
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP User _name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:424
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:457
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Start"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:494
msgid "Discovering iSCSI targets.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:534
msgid "Discovery login rejected."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:549
msgid ""
"The following error occurred discovering iSCSI targets.  Please double check "
"your authorization information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:571
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Retry"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:611
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Bind targets to network interfaces This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:686
msgid "Node Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:697
msgid "Interface"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:708
msgid "Portal"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:737
msgctxt "GUI|Advanced Storage|iSCSI|Login"
msgid "_Node Login Authentication Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:754
msgid "Use the credentials from discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:789
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:803
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:909
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:923
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _Password:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:937
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "_Reverse CHAP User name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:951
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:999
msgctxt "GUI|Advanced Storage|iSCSI|Login|Login"
msgid "_Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1055
msgid "Node login failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1070
msgid ""
"The following error occurred logging into the selected iSCSI node.  Please "
"double check your authorization information and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1092
msgctxt "GUI|Advanced Storage|iSCSI|Login|Retry"
msgid "Retry _Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:22
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:36
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:68
msgid "ADD DASD STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:97
msgid "To use DASD disks, you must provide the device number."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:125
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Device number:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:145
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:182
msgid "Discovering DASD devices.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:224
#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:267
msgid "Device discovery failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:241
msgid ""
"The following error occurred discovering DASD devices.  Please double check "
"your configuration information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:267
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:82
msgid "ADD zFCP STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:111
msgid ""
"To use zFCP disks, you must provide the device number, WWPN, and LUN "
"configured for the device."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:139
msgctxt "GUI|Advanced Storage|zFCP|Device Number"
msgid "_Device number:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:168
msgid "WWPN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:188
msgctxt "GUI|Advanced Storage|zFCP|Start Discovery"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:225
msgid "Discovering zFCP devices.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:284
msgid ""
"The following error occurred discovering zFCP devices.  Please double check "
"your configuration information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:310
msgctxt "GUI|Advanced Storage|zFCP|Retry Discovery"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:356
msgid "LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:26
#: pyanaconda/ui/gui/spokes/lib/cart.glade:78
msgid "SELECTED DISKS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:43
msgctxt "GUI|Selected Disks Dialog"
msgid "_Close"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:106
msgid "Boot"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:120
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:222
msgid "Description"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:158
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:261
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:739
msgid "Free"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:199
msgctxt "GUI|Selected Disks Dialog"
msgid "_Remove"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:226
msgid "Disk summary goes here"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:21
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:37
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Delete It"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:129
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:144
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Select"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:176
msgid "CONFIGURE MOUNT POINT"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:191
msgid "Select one or more disks this device may reside on."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:321
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:336
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Add mount point"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:369
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Desired Capacity:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:386
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "Mount _Point:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:404
msgid "ADD A NEW MOUNT POINT"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:421
msgid ""
"More customization options are available\n"
"after creating the mount point below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:456
msgid ""
"That mount point is already in\n"
"use.  Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:480
msgid "eg: \"20 GB\", \"500mb\" (minus the quotation marks)"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:535
msgid "Single <span foreground=\"grey\">(No Redundancy, No Striping)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:583
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:597
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Save"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:629
msgid "CONFIGURE CONTAINER"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:644
msgid ""
"Please create a name for this container and select at least one disk below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:662
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:781
msgid "RAID Level:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:809
msgid "Encrypt"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:850
msgid "Please enter a valid name."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:880
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "Si_ze policy:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:895
msgid "Automatic"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:896
msgid "As large as possible"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:897
msgid "Fixed"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:8
msgid "UNFORMATTED DASDS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:23
msgctxt "GUI|DAD Format Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:37
msgctxt "GUI|DAD Format Dialog"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:70
msgid ""
"The following unformatted DASDs have been detected on your system. You can "
"choose to format them now with dasdfmt or cancel to leave them unformatted. "
"Unformatted DASDs cannot be used during installation."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:86
msgid "Unformatted DASDs Detected"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:136
msgid ""
"<b>Warning</b>: All storage changes made using the installer will be lost "
"when you choose to format."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:160
msgctxt "GUI|DAD Format Dialog"
msgid "_Format with dasdfmt"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:207
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while formatting completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:247
msgid "Disk formatting complete."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/detailederror.glade:25
msgid "An unknown error occurred during installation.  Details are below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:23
msgctxt "GUI|Passphrase Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:38
msgctxt "GUI|Passphrase Dialog"
msgid "_Save Passphrase"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:66
msgid "DISK ENCRYPTION PASSPHRASE"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:82
msgid ""
"You have chosen to encrypt some of your data. You will need to create a "
"passphrase that you will use to access your data when you start your "
"computer."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:103
msgctxt "GUI|Passphrase Dialog"
msgid "_Passphrase:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:132
msgctxt "GUI|Passphrase Dialog"
msgid "Con_firm:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:270
msgid ""
"Warning: You won't be able to switch between keyboard layouts (from the "
"default one) when you decrypt your disks after install."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:8
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:71
msgid "RESCAN DISKS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:25
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:39
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_OK"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:87
msgid ""
"You can remove or insert additional disks at this time and press 'Rescan "
"Disks' below for the changes to take effect."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:118
msgid ""
"<b>Warning:</b> All storage changes made using the installer will be lost "
"when you press 'Rescan Disks'."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:143
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Rescan Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:176
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:212
msgid "Scanning disks.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:223
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while this scan completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:254
msgid "Disk rescan complete."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:269
msgid ""
"Pressing 'OK' below will take you to the disk selection screen where you "
"will need to re-select your disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:55
msgctxt "GUI|Reclaim Dialog"
msgid "_Cancel"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:70
msgctxt "GUI|Reclaim Dialog"
msgid "_Reclaim space"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:104
msgid "RECLAIM DISK SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:121
msgid "Description goes here."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:174
msgid "File System"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:185
msgid "Reclaimable Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:196
#: pyanaconda/ui/gui/spokes/lib/summary.glade:144
msgid "Action"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:221
msgctxt "GUI|Reclaim Dialog"
msgid "_Preserve"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:237
msgctxt "GUI|Reclaim Dialog"
msgid "_Delete"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:253
msgctxt "GUI|Reclaim Dialog"
msgid "_Shrink"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:367
msgid "Reclaim"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:25
#: pyanaconda/ui/gui/spokes/lib/summary.glade:89
msgid "SUMMARY OF CHANGES"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:42
msgctxt "GUI|Summary Dialog"
msgid "_Cancel & Return to Custom Partitioning"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:57
msgctxt "GUI|Summary Dialog"
msgid "_Accept Changes"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:105
msgid ""
"Your customizations will result in the following changes taking effect after "
"you return to the main menu and begin installation:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:133
msgid "Order"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:166
msgid "Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:177
msgid "Mount point"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:35
msgid ""
"The system needs better quality of random data, you can improve it by typing "
"randomly on keyboard and moving your mouse. The installation will continue "
"automatically regardless of random data quality when time runs out."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:53
msgid "Random data quality:"
msgstr ""

#: pyanaconda/ui/gui/main.glade:94
msgctxt "GUI|Main"
msgid "_No"
msgstr ""

#: pyanaconda/ui/gui/main.glade:108
msgctxt "GUI|Main"
msgid "_Yes"
msgstr ""

#: pyanaconda/ui/gui/main.glade:151
msgid ""
"Are you sure you wish to quit the\n"
"installation process?"
msgstr ""

#: pyanaconda/ui/gui/main.glade:175
msgid "Quit"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade:8
#: pyanaconda/ui/gui/hubs/summary.glade:147
msgid "INSTALLATION SUMMARY"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade:84
msgctxt "GUI|Summary"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade:100
msgctxt "GUI|Summary"
msgid "_Begin Installation"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade:131
msgid "We won't touch your disks until you click 'Begin Installation'."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade:8
#: pyanaconda/ui/gui/hubs/progress.glade:261
msgid "CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade:91
msgid "Preparing to install"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade:149
msgctxt "GUI|Progress"
msgid "_Finish configuration"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade:202
msgctxt "GUI|Progress"
msgid "_Reboot"
msgstr ""

#: data/liveinst/liveinst.desktop.in.h:1
msgid "Install to Hard Drive"
msgstr ""

#: data/liveinst/liveinst.desktop.in.h:2
msgid "Install"
msgstr ""

#: data/liveinst/liveinst.desktop.in.h:3
msgid "Install the live CD to your hard disk"
msgstr ""

#: /home/bcl/Red_Hat/projs/anaconda/data/liveinst/console.apps/liveinst:5
msgid "Starting Install to Hard Drive"
msgstr ""

#: data/liveinst/gnome/fedora-welcome.desktop.in.h:1
#: data/liveinst/gnome/fedora-welcome:65
msgid "Welcome to Fedora"
msgstr ""

#: anaconda:467
#, python-format
msgid ""
"%(product_name)s requires %(needed_ram)s MB of memory to install, but you "
"only have %(total_ram)s MB on this machine.\n"
msgstr ""

#: anaconda:469
#, python-format
msgid ""
"The %(product_name)s graphical installer requires %(needed_ram)s MB of "
"memory, but you only have %(total_ram)s MB\n"
"."
msgstr ""

#: anaconda:472
msgid ""
"\n"
"Press <return> to reboot your system.\n"
msgstr ""

#: anaconda:474
msgid "Not enough RAM"
msgstr ""

#: anaconda:475
msgid ""
" Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""

#: anaconda:478
msgid " Starting text mode."
msgstr ""

#: anaconda:515
msgid "Fatal Error"
msgstr ""

#: anaconda:640
msgid "DISPLAY variable not set. Starting text mode."
msgstr ""

#: anaconda:648
msgid "Graphical installation is not available. Starting text mode."
msgstr ""

#: anaconda:656
msgid ""
"Text mode provides a limited set of installation options. It does not offer "
"custom partitioning for full control over the disk layout. Would you like to "
"use VNC mode instead?"
msgstr ""

#: anaconda:746
#, python-format
msgid "Please ssh install@%s to begin the install."
msgstr ""

#: anaconda:748
msgid "Please ssh install@<host> to continue installation."
msgstr ""

#: anaconda:989
msgid "Unable to create PID file"
msgstr ""

#: anaconda:990
#, python-format
msgid ""
"Anaconda is unable to create %s because the file already exists. Anaconda is "
"already running, or a previous instance of anaconda has crashed."
msgstr ""

#: data/liveinst/gnome/fedora-welcome:94
msgid "Try Fedora"
msgstr ""

#: data/liveinst/gnome/fedora-welcome:110
msgid ""
"You are currently running Fedora from live media.\n"
"You can install Fedora now, or choose \"Install to Hard Drive\" in the "
"Activities Overview at any later time."
msgstr ""

#: data/liveinst/gnome/fedora-welcome:128
msgid ""
"You can choose \"Install to Hard Drive\"\n"
"in the Activities Overview at any later time."
msgstr ""

#: data/liveinst/gnome/fedora-welcome:131
msgid "Close"
msgstr ""
